### PR TITLE
feat: Onboarding new repositories (UP 12.1 and UP 12.2)

### PR DESCRIPTION
**Pull request by msampathkumar**
Onboarding UP 12 repositories.

* Golang -  https://github.com/googlecloudplatform/terraform-pubsub-integration-golang
* Java - https://github.com/googlecloudplatform/terraform-pubsub-integration-java

--
Raised a intake request at http://b/298414847

### CODE REVIEW

File: infra/terraform/test-org/org/locals.tf [hunk 1]
```diff
@@ -752,5 +752,23 @@ locals {
       owners      = ["anaik91", "imrannayer"]
       topics      = local.common_topics.db
     },
+    {
+      name        = "terraform-pubsub-integration-golang"
+      short_name  = "pubsub-golang-app"
+      org         = "GoogleCloudPlatform"
+      description = "Reliable low-latency server-to-server communication around the globe"
+      owners      = ["anguillanneuf", "balajismaniam", "donmccasland", "msampathkumar"]
+      # groups          = ["dee-data-ai", local.jss_common_group]
```
Comment (g-awmalik) [inline infra/terraform/test-org/org/locals.tf:761]: Please take these out if not needed. Although we've been enabling periodic testing for all JSS solutions so you may want to keep that.

Same comments for the repo below.

File: infra/terraform/test-org/org/locals.tf [hunk 2]
```diff
@@ -752,5 +752,17 @@ locals {
       owners      = ["anaik91", "imrannayer"]
       topics      = local.common_topics.db
     },
+    {
+      name            = "terraform-pubsub-integration-golang"
+      org             = "GoogleCloudPlatform"
+      short_name      = "pubsub-golang-app"
```
Comment (donmccasland) [inline infra/terraform/test-org/org/locals.tf:758]: both of these need owners and groups.  Example:
      owners      = ["shabirmean", "Mukamik"]
      groups      = ["dee-platform-ops", local.jss_common_group]

Comment (msampathkumar) [reply infra/terraform/test-org/org/locals.tf:758]: thank you @donmccasland !

**Review by g-awmalik — APPROVED** [review]

**Review by donmccasland — APPROVED** [review]

**Comment by g-awmalik** [comment]
@msampathkumar - can you add [blueprint-solutions](https://github.com/orgs/GoogleCloudPlatform/teams/blueprint-solutions) as admin to both these repos?

**Comment by msampathkumar** [comment]
Hi @g-awmalik updated the config. Please check & suggest. Thank you.

**Comment by g-awmalik** [comment]
> Hi @g-awmalik updated the config. Please check & suggest. Thank you.

I'm sorry I should've been more clear. I mean add blueprint-solutions to the Github repos i.e .https://github.com/googlecloudplatform/terraform-pubsub-integration-golang & https://github.com/googlecloudplatform/terraform-pubsub-integration-java. We need admin right to run the TF configs that update them and add them to our CI triggers.

**Comment by msampathkumar** [comment]
Hi @g-awmalik thank you. I have provided the admin access to these repos.

**Pull request merged**